src/viewer/viewer.ts

```diff
@@ -1148,9 +1149,11 @@ export default class Viewer {
   }
 
   private __renderPickingGroup (camera: PerspectiveCamera|OrthographicCamera) {
-    this.renderer.clearTarget(this.pickingTarget, true, true, true)
+    this.renderer.setRenderTarget(this.pickingTarget || null!)
```

Comment:
It looks like the `!` indicator can now be removed as `null` is an accepted argument in three.js typings. Same applies for l:1156 below

Reply:
Yup, looks like they fixed that. Thanks!